sql/logic_test.go

```diff
@@ -202,8 +202,14 @@ func (t *logicTest) setUser(user string) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	// TODO(tamird): SSL
-	db, err := sql.Open("postgres", fmt.Sprintf("sslmode=disable user=%s host=%s port=%s", user, host, port))
+	dir, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	certDir := filepath.Join(filepath.Dir(dir), "resource", security.EmbeddedCertsDir)
+	db, err := sql.Open("postgres",
+		fmt.Sprintf("sslmode=verify-full sslrootcert=%s user=%s host=%s port=%s",
```

Comment:
Shouldn't this fail because the client isn't presenting a certificate to the server? I think after calling `tls.Server`, you need to call `conn.Handshake` to wait for the connection to be established, then inspect `Conn.ConnectionState()` (at least the `PeerCertificates` field - look at what the http/rpc server does for this)